dashboard/app/models/pd/international_opt_in.rb

```diff
@@ -12,11 +12,18 @@
 #
 #  index_pd_international_opt_ins_on_user_id  (user_id)
 #
+require 'json'
 
 class Pd::InternationalOptIn < ApplicationRecord
   include Pd::Form
   include InternationalOptInPeople
 
+  COLOMBIAN_SCHOOL_DATA = JSON.parse(
+    File.read(
+      File.join(Rails.root, 'config', 'colombianSchoolData.json')
```

Comment:
can we use a shorter dataset `if CDO.stub_school_data` ?

Reply:
we _could_ but I'm not convinced we _should_. My understanding is that option is used to change how we seed our data on US schools specifically, it's not clear to me that it has relevance here. Do you have a specific reason why you'd like to see that change?

Reply:
yes - the point is to avoid slowing down the application startup time in environments (currently development, adhoc and test, including drone) which do not need the full file. IIRC `dashboard/config/schools.tsv` takes about 60 seconds to seed and is only 3-4x larger than this file, so I think it's worth at least checking that loading and parsing this file doesn't add more than a few seconds to application startup time.

Reply:
Oh, that makes sense!

I put together this test:

```ruby
#!/usr/bin/env ruby

require 'json'

start = Time.now
COLOMBIAN_SCHOOL_DATA = JSON.parse(
  File.read(
    File.join("dashboard", "config", "colombianSchoolData.json")
  )
).freeze
finish = Time.now

puts "JSON loaded in #{finish-start} seconds"
```

the result on my local machine:

```
JSON loaded in 0.052831755 seconds
```

I'd conclude that the time added by this is negligible, and we should be fine to deploy without a flag.

Reply:
LOL, that seems within the time budget :) thanks for checking! maybe all the DB operations make seeding schools take forever.